pcp/PCPDynamicMeter.c

```diff
@@ -197,7 +197,7 @@ static void PCPDynamicMeter_parseFile(PCPDynamicMeters* meters, const char* path
          if (ok)
             meter = PCPDynamicMeter_new(meters, key + 1);
       } else if (!ok) {
-         ;  /* skip this one, we're looking for a new header */
+         (void)0;  /* skip this one, we're looking for a new header */
       } else if (value && meter && String_eq(key, "caption")) {
```

Comment:
Code style note:

Adding a check `else if (!value || !meter) { (void)0; }` can simplify the conditions below.

Change similar to what is already present in the other location affected by this change.